benchmark/fio_benchmark_cleanup.yml

```diff
@@ -38,29 +38,19 @@
     - name: Delete image
       command: "rbd remove {{ fiobench_pool_name | default('fiobench') }}/{{ item }} -k /etc/ceph/ceph.client.fiobench.keyring -n client.fiobench"
       with_items:
-        - "bench_file"
-        - "bench_direct"
+        - "bench_file-{{ inventory_hostname_short }}"
+        - "bench_direct-{{ inventory_hostname_short }}"
 
 # Cleanup the benchmark pool if we created it
 - hosts: mons[0]
   become: True
   tasks:
-    - name: Check the value of mon_allow_pool_delete
```

Comment:
In luminous you can't delete a pool without setting the mon_allow_pool_delete flag to true, so we probably need this.

It also ensures the setting is returned back to false if it was originally false, but is left true if it was already true for consistency.